bubble_sort_by.rb

```diff
@@ -0,0 +1,22 @@
+def bubble_sort(array)
```

Comment:
If this is `bubble_sort_by`, let it reflect in the naming. Also, you could just create both methods in one file.

Reply:
Thanks @suretrust. Your suggestions will be enacted. 👍 🙏